TLM/TMPE.API/Traffic/Data/SpeedValue.cs

```diff
@@ -122,6 +122,14 @@ public KmphValue ToKmphPrecise()
         public static SpeedValue operator +(SpeedValue left, SpeedValue right)
             => new(left.GameUnits + right.GameUnits);
 
+        public static bool operator ==(SpeedValue left, SpeedValue right) {
+            return left.Equals(right);
```

Comment:
this.Equals is small enough to fully copy here, and not call another function. Will compiler inline this call to Equals?

Reply:
No idea, that equality checks code were generated by Rider. I can change 👍